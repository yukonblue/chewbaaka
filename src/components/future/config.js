/**
 * future/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 14, 2020
 * Updated  : Jun 10, 2022
 */

export const config = {
  headMeta: {
    title: "Future of the cheetah | Learn about cheetahs",
    description: "Learn about the future of the cheetahs, and conservation actions for saving them from extinction.",
    keywords: [
      "cheetah status",
      "cheetah future",
      "illegal cheetah trafficking",
      "livestock guarding dog",
    ]
  },
  pageProps: {
    coverImageNamePrefix: "Future_Page_Cover_Image",
    title: "Future",
    subtitle: "Learn about CCF's holistic approach to conservation and sustainable development, and what you can do to help.",
    pageMenuItems: [
      {
        title: "Conservation",
        tocImageFilename: "future_page_toc_menu_img_01-min.jpg"
      },
      {
        title: "Sustainable Development",
        tocImageFilename: "future_page_toc_menu_img_02-min.jpg"
      },
      {
        title: "Outreach and Education",
        tocImageFilename: "future_page_toc_menu_img_03-min.jpg"
      }
    ],
    pageTailNavMenu: {
      prevPage: {
        label: "Ecology",
        href: "/ecology/"
      }
    }
  },
  contentPageIntro: {
    title: "Future of the Cheetah",
    content: "With only 7,500 individuals of cheetahs left in the wild, the species is facing unprecedented pressures " +
            "that are pushing its fate to the brink of extinction. Needless to say, the cheetah’s future lies in our hands. " +
            "Despite the grim facts, we hold a positive outlook on their future. A number of nation governments, " +
            "non-profit organizations, as well as individuals have pulled themselves together to combine their " +
            "ingenuity and resourcefulness to help the cheetahs fight for a brighter future. We believe that anyone " +
            "who is passionate in the cause has something to contribute, and we can only come out triumphantly if we " +
            "work together to protect the cheetahs, nurture their ecosystems, and make ourselves better custodians of " +
            "this planet and all living things that share it with us.",
    image: {
      filenamePrefix: "Future_Page_Intro_Section_Background",
      aspectRatio: {
        width: 640,
        height: 640
      }
    }
  },
  contentPageSections: {
    section_Conservation: {
      intro: {
        title: "Conservation",
        content: "Successful conservation approaches need to be holistic and address the diverse array of inter-related issues, " +
                "in order to make sure that all parties involved benefit, including the species to be conserved, their ecosystem, " +
                "and any sustainable utilizations of resources within the ecosystem. Cheetah Conservation Fund develops a set of " +
                "conservation strategies to ensure that best practices of wildlife and farm management are employed to benefit the " +
                "cheetahs and their habitats, as well as Namibians who share the same land and resources."
      },
      bgColor: "#F4EBDB",
      subsections: {
        subsection_WhatIsConservation: {
          title: "What is Conservation?",
          contents: {
            part_Intro: {
              content: {
                paragraph_01:
                  "Conservation is taking care of the environment in which we live. This means using everything wisely " +
                  "so some of it is left for others to use or for our own future use. The things we use are called RESOURCES. " +
                  "These can be almost anything: food, the air we breathe, firewood, petrol and zinc footing panels. " +
                  "Conservationists understand that RESOURCES ARE LIMITED and often scarce and that we must use them wisely. " +
                  "In Namibia, fresh water is one of our scarcest resources.",
                paragraph_02:
                  "Conservation also means protecting resources we may not use.",
                paragraph_03:
                  "Walking around, rather than stepping on a plant, is an act of conservation.",
                paragraph_04:
                  "Protecting a species like the cheetah from extinction is an act of conservation."
              }
            },
            part_EnvironmentalEducation: {
              title: "Environmental Education",
              content: {
                paragraph_01:
                  "Environmental education encourages the wise use of natural resources including water and land use, waste management and recycling."
              }
            },
            part_ConservationBiology: {
              title: "Conservation Biology",
              content: {
                paragraph_01:
                  "Conservation Biology is a field of study at universities. It combines several sciences, such as biology, " +
                  "ecology and genetics, to examine why there are so many types of living things on Earth and what makes some of them scarce."
              }
            }
          }
        },
        subsection_StoppingIllegalWildlifeTrades: {
          title: "Stopping Illegal Wildlife Trades",
          subtitle: "The illegal trafficking of cheetahs into the pet trade has become one of the main threats to cheetah survival.",
          contents: {
            part_CCFInvolvementPart1: {
              content: {
                paragraph_01:
                  "Since 2005, CCF has been actively involved with issues involving the illegal trade of live cheetah cubs. " +
                  "Staff monitor cheetah trafficking activities and organizes confiscations through proper authorities when possible.",
                paragraph_02:
                  "Cheetahs are listed as an Appendix I species under the Convention on International Trade in Endangered Species (CITES). " +
                  "They are poached from the wild for the pet trade and their body parts, which get sold on the markets. " +
                  "Illegal wildlife trafficking is one of the top five transnational crimes, generating an estimated U.S. $50-150 million " +
                  "in illegal revenues annually.",
                paragraph_03:
                  "CCF is taking a leadership role in addressing the illegal pet trade. CCF trains wildlife officers involved " +
                  "in the front lines of trafficking in the Horn of Africa as well as in the Middle East, " +
                  "teaching them proper care for confiscated animals.",
                paragraph_04:
                  "CCF has built the most extensive database on cheetah trafficking, recording hundreds of trafficking cases going through the Horn of Africa to the Middle East."
              }
            },
            part_CITESCoP17: {
              content: {
                paragraph_01:
                  "CCF participates in the CITES inter-sessional working group on the illegal trade in cheetahs, and was instrumental " +
                  "in making recommendations adopted by CITES at CoP17 to stop the illegal cheetah pet trade."
              }
            }
          }
        },
        subsection_CheetahAmbassadors: {
          title: "Cheetah Ambassadors",
          contents: {
            part_Intro: {
              content: {
                paragraph_01:
                  "Cheetahs in captivity can serve as ambassadors that can help in spreading the message of " +
                  "conservation and sustainable development in education and outreach programs. " +
                  "Here at Cheetah Conservation Fund, many cheetahs that had been cared by CCF have left a legacy for future generations."
              }
            },
            part_Chewbaaka: {
              content: {
                paragraph_01:
                  "Hi there, my name is Chewbaaka. I was named after an animal " +
                  "in a movie. I have lived at CCF since I was 3 weeks old. " +
                  "I am their cheetah ambassador and I get to meet many people. " +
                  "The staff at CCF tell everyone about me and all the wild " +
                  "cheetahs. I have a special place where I can run like the wind.",
                paragraph_02:
                  "Chewbaaka played a big part in CCF’s rewilding program by " +
                  "helping farmers understand more about how special cheetahs " +
                  "are and that their survival is in the farmers’ hands. He brought a " +
                  "calming presence to the Namibian farmers he met when they " +
                  "visited CCF. During Chewbaaka’s 16 years, he met thousands of " +
                  "farmers and school learners.",
                paragraph_03:
                  "Without farmer support, CCF wouldn't be able to release " +
                  "cheetahs back into the wild. By helping people live alongside " +
                  "predators CCF is helping save the cheetah and its ecosystem."
              },
              card_content: "Chewbaaka was a parnter of Dr. Laurie Marker and together they have worked to educate and inspire tens of thousands of children and farmers across Namibia. Chewbaaka was a living demonstration that wildlife and people can coexist together peacefully and harmoniously."
            },
            part_Mekondyo: {
              content: {
                paragraph_01:
                  'Hello, my name is Mekondyo. Mekondyo means "struggle" in the ' +
                  "Oshiwambo language. I was born on a farm north of Otjiwarongo " +
                  "but I now live on farmland west of Otjiwarongo. It is very " +
                  "beautiful. I can see the Waterberg Plateau far away. " +
                  "I am 5 years old now and I will tell you my story as you " +
                  "explore this museum."
              },
              card_content: 'Mekondyo means "struggle" in the Oshiwambo language.'
            } 
          }
        },
      },
    },
    section_SustainableDevelopment: {
      intro: {
        title: "Sustainable Development",
        content: "Sustainable development is key to ensure long term utilizations of natural resources without compromising the integrity " +
                "of the ecosystem and the sharing of resources by other species. Part of sustainable development involves taking part of " +
                "the revenue generated from any human development and contribute that to the nurturing and protection of the entire ecosystem. " +
                "Any human development can be turned to be sustainable by employing the best practices of land, livestock, and wildlife management " +
                "techniques that have proven to be effective and successful."
      },
      bgColor: "#F1F0FF",
      subsections: {
        subsection_MissionPossible: {
          title: "Mission Possible",
          contents: {
            part_Intro: {
              content: {
                paragraph_01:
                  "In Africa, property that is owned and managed by farmers can maintain viable populations of " +
                  "animals and natural habitats. It is the careful management of these habitats that holds the key " +
                  "to the future survival of plant and animal species such as the cheetah.",
                paragraph_02:
                  "In order to ensure a successful future we need to be responsible custodians of nature. " +
                  "The way land and animals are managed determines the future of all ecosystems."
              }
            },
            part_Body: {
              content: {
                part_LandManagement: {
                  title: "Land Management",
                  content: {
                    paragraph_01:
                      "The productivity of the land depends on water and soil. Preventing erosion increases the potential " +
                      "for the land to produce food and allows water to penetrate the soil, thus carrying important nutrients " +
                      "to the roots of the plants. We must learn to live within the limits of the scarcity of water in our dry country.",
                    paragraph_02:
                      "The use of alternative fuels or efficient wood burning stoves and ovens reduces deforestation. " +
                      "Establishing timber plantations and using alternative building materials saves natural forests."
                  }
                },
                part_LivestockManagement: {
                  title: "Livestock Management",
                  content: {
                    paragraph_01:
                      "Many farmers use rotational grazing where farmlands are divided into camps. Practicing rotational and " +
                      "seasonal grazing allows used areas time to recover.",
                    paragraph_02:
                      "Recovery of damaged land occurs faster if it is protected from the impact of overgrazing. " +
                      "Proper herd size is necessary to prevent overgrazing, trampling of the land and reducing predator problems. " +
                      "A balance of grazing and browsing animals reduces the pressure on one vegetation type."
                  }
                },
                part_WildlifeManagement: {
                  title: "Wildlife Management",
                  content: {
                    paragraph_01:
                      "Allowing wild animals to migrate naturally through farm areas promotes the balance of browsers and grazers, " +
                      "and allows predators their food at wild game. This reduces the temptation to take domestic stock.",
                    paragraph_02:
                      "Ethical hunting removes older animals while predators prey on the young, diseased, and genetically weak. " +
                      "Together they keep wild herds healthy and in check."
                  }
                }
              }
            }
          }
        },
        subsection_FutureFarmersOfAfrica: {
          title: "Future Farmers of Africa",
          contents: {
            paragraph_FutureFarmersOfAfrica_01:
              "In Namibia, 90% of the cheetah population lives on farmlands alongside 80% of the country’s wildlife species. " +
              "Cheetahs often come into contact with the livestocks and game farming communities, leading to conflict. " +
              "To mitigate this conflict, Cheetah Conservation Fund (CCF) developed an integrated livestock, wildlife, and " +
              "rangeland training program called Future Farmers of Africa (FFA).",
            paragraph_FutureFarmersOfAfrica_02:
              "From 2006-2017, CCF has certified more than 10,000 Namibian men and women in FFA, which teaches conservation " +
              "strategies for sustainable land use while accommodating the coexistence of predators alongside livestock and wildlife.",
            paragraph_FutureFarmersOfAfrica_03:
              "Topics covered in the FFA courses include livestock health and veterinary care, livestock husbandry, livestock valuation, " +
              "predator spoor (track) identification, differentiating predator kill techniques, best practices to reduce livestock losses, " +
              "and best methods for non-lethal predator control.",
            paragraph_FutureFarmersOfAfrica_04:
              "Through the training course, CCF supports farmers by teaching them practical skills in sustainable livestock farming, " +
              "as well as how to develop supplemental income streams. Farmers then use their skills to generate higher economic productivity. " +
              "These courses enhance the livelihood development of farmers while promoting predator-friendly farming practices.",
            paragraph_FutureFarmersOfAfrica_05:
              "CCF’s Integrated Livestock and Wildlife Model Farm houses cattle, goats, sheep and livestock guarding dogs, as well as Namibian wildlife. " +
              "The Model Farm serves as a training facility for FFA, demonstrating how sustainable livestock farming practices allow for the " +
              "management of their livestock and better livestock health without having to eliminate cheetahs and other predators.",
            paragraph_FutureFarmersOfAfrica_06:
              "CCF reaches out to local farming communities, visiting farms and joining annual farmer’s association meetings to give vital " +
              "advice on improvements that farmers can make to their rangeland, livestock and wildlife management."
          }
        },
        subsection_LivestockGuardingDogs: {
          title: "Livestock Guarding Dogs",
          contents: {
            part_LGD_History: {
              title: "CCF’s Livestock Guarding Dog History",
              content: {
                paragraph_01:
                  "In 1994, CCF created a solution to non-lethal predator control - Livestock Guarding Dogs (LGD). " +
                  "After extensive research, CCF felt that the Anatolian Shepherd and Kangal dogs best met the livestock managers’ needs in Namibia. " +
                  "In cooperation with Dr. Ray Coppinger from the Livestock Guarding Dog Association in Hampshire College (Massachusetts, U.S.), " +
                  "the first 10 dogs were donated to Namibia by Birinci Anatolians.",
                paragraph_02:
                  "Since then, hundreds of farmers have received Anatolian Shepherds and Kangal dogs to ward off predators, such as cheetahs, jackals and leopards. " +
                  "These dogs do not herd the goats, but protect the flock by placing themselves between the herd and the predator and barking loudly.",
                paragraph_03:
                  "CCF’s success is due to the commitment of CCF staff members and the participating farmers. CCF staff regularly visit the farms, " +
                  "educating and advising the farmers and herders for the dogs to be as effective as possible."
              }
            },
            part_Raising_LGDs: {
              title: "Raising LGD’s",
              content: {
                paragraph_01:
                  "LGD’s are placed with their herds around nine weeks of age, as social bonding between the stock and the dogs deepens upon " +
                  "imprinting before 16 weeks of age. The dogs live with their livestock family instead of a human family, so the puppies bond with the herd; " +
                  "it becomes the dog’s pack rather than the dog’s potential prey. At CCF puppies are born in a whelping pen attached to the main goat yard."
              }
            },
            part_Dogs_Save_Cheetahs: {
              title: "Dogs Save Cheetahs",
              content: {
                paragraph_01:
                  "Livestock Guarding Dogs have been used for thousands of years to protect cattle, sheep and goats from predator attacks. " +
                  "There are over 20 breeds of guarding dogs. The Anatolian Shepherd, a Turkish breed, was selected as the best candidate for " +
                  "use with smallstock in Namibia as they are able to work in vast open spaces without direct guidance. " +
                  "These dogs look similar to the flock, with large rounded heads, floppy ears and short fur. " +
                  "They have a good sense of hearing and smelling, a calm temperament and a very loud bark.",
                paragraph_02:
                  "Anatolians do not herd livestock, they guard them. They are attentive, protective, trustworthy, and aggressive towards predator threats.",
                paragraph_03:
                  "Puppies are placed with the stock at eight weeks of age to form a strong bond with the herd. " +
                  "It is important that bonding occurs with the herd and not with humans or other dogs.",
                paragraph_04:
                  "Dog owners take the responsibility for the health care of their dogs. This includes veterinary check-ups, vaccinations, " +
                  "and observations for signs of illness or injuries.",
                paragraph_05:
                  "An appropriate diet is necessary for the Anatolian Shepherd. Enough food allows proper growth and a healthy dog. " +
                  "They should never be given raw meat as this could produce a predatory response. " +
                  "They need water during the day when they are out with the herd."
              }
            }
          }
        }
      }
    },
    section_OutreachAndEducation: {
      intro: {
        title: "Outreach and Education",
        content: "Public education is a vital component of any successful conservation strategies. " +
                "Environmental and management education are critical in areas where people coexist intimately with the land and wildlife. " +
                "Cheetah Conservation Fund believes public education and outreach will help ensure a future for the species in the wild. " +
                "CCF educates farmers, teachers, students, and the public about methods to conserve biodiversity, " +
                "as well as champion the idea that everybody can be a custodian of nature and has something to contribute."
      },
      bgColor: "#CDFFFF",
      subsections: {
        subsection_FieldResearch: {
          title: "Field Research",
          content: {
            part_VeterinaryClinic: {
              title: "Veterinary Clinic",
              content: {
                paragraph_01:
                  "The HASS Family Research Center houses CCF’s registered on-site veterinary clinic. " +
                  "The clinic allows CCF to provide medical care for cheetahs within the sanctuary, " +
                  "CCF Livestock Guarding Dogs and CCF’s livestock. In addition, all cheetahs that CCF studies " +
                  "receive a full biomedical work-up and health assessment.",
                paragraph_02:
                  "Other clinical procedures include routine medical checks, dental care and when necessary, surgery. " +
                  "While working with these animals, CCF research staff can collect biological samples that are used to do health evaluation, " +
                  "reproduction studies and to determine the cheetah population’s genetic status and provide the basis of a disease surveillance " +
                  "system for cheetahs, these biological samples also aid in future research.",
                paragraph_03:
                  "Wild cheetahs that come into the clinic are marked with a transponder ear tag and/or fitted with a radio or " +
                  "satellite tracking collar before release back into the wild. CCF works with the Namibian Ministry of " +
                  "Environment and Tourism and neighboring farmers to determine a suitable habitat for relocation if necessary, " +
                  "or release on the farm where the cheetah was caught."
              }
            },
            part_EcologicalStudies: {
              title: "Ecological Studies",
              content: {
                paragraph_01:
                  "Satellite tracking collars provide important data for CCF research teams. " +
                  "Researchers can monitor the daily movement of the cheetah, track its hunting skills and " +
                  "monitor its general well-being in the wild. If also allows researchers to evaluate the " +
                  "cheetah’s habitat use and prey preference.",
                paragraph_02:
                  "Satellite and VHS monitoring collars, as well as camera traps help determine animal home ranges, " +
                  "habitat preferences and seasonal use, territoriality and behaviors unique to individual cheetah populations.",
                paragraph_03:
                  "CCF conducts vegetation studies that include monitoring growth patterns of bush within targeted study areas on CCF land. " +
                  "Information on vegetation is used to identify target areas for ecological management and help determine and monitor the " +
                  "manner with which bush encroachment affects the biodiversity of the land."
              }
            }
          }
        },
        subsection_InternshipsAndVolunteering: {
          title: "Internships and Volunteering",
          content: {
            part_Intro: {
              content: {
                paragraph_01:
                  "Cheetah Conservation Fund’s commitment to scientific research has helped grow the next generation of conservation " +
                  "scientists and environmentalists."
              }
            },
            part_Internships: {
              title: "Internships",
              content: {
                paragraph_01:
                  "CCF welcomes a selected number of students each year with a wide variety of backgrounds and courses of study. " +
                  "Graduate and undergraduate students are given a chance to learn about aspects of conservation biology, ecology, " +
                  "veterinary medicine, genetics, animal science, agriculture, sociology, business, computing, graphic design, education and tourism.",
                paragraph_02:
                  "CCF is an official fourth year placement for students from Namibia’s two major universities, " +
                  "University of Namibia (UNAM) and Namibia University of Science and Technology (NUST). " +
                  "In addition, CCF welcomes interns from undergraduate and graduate university programs all over the world.",
                paragraph_03:
                  "Aspiring biologists, genetics, and ecologists pursuing Masters and Ph.D. degrees come to CCF to work on research and thesis projects all year."
              },
              internship_list_items: [
                "cheetah feeding - collecting and dissecting cheetah feces",
                "observing cheetah behavior - assisting in clinical work-ups",
                "raise livestock guarding dog puppies - conducting field work",
                "being involved with outreach and education programs"
              ]
            },
            part_Volunteering: {
              title: "Volunteering",
              content: {
                paragraph_01:
                  "Volunteering at CCF in Namibia is a rewarding experience, and most of the people who participate in our volunteer " +
                  "program will tell you that it is among the best experiences they have ever had. Volunteers do everything from data entry " +
                  "and camera trap photo sorting to chopping up meat to collecting and cataloging scat samples.",
                paragraph_02:
                  "Like interns, volunteers can expect to work on a variety of tasks. CCF also seeks professionally skilled volunteers to " +
                  "work as cheetah keepers, animal behavior specialists, ecologists, biologists, veterinarians, and vet-technicians, " +
                  "as well as educators, trainers, and conservationists."
              }
            }
          }
        },
        subsection_TheRoleOfZoos: {
          title: "The Role of Zoos",
          contents: {
            part_Intro: {
              content: {
                paragraph_01:
                  "Moderns zoos serve as conservation education centers and provide sanctuary for many endangered species. " +
                  "Cooperative breeding, management and research have assisted in re-establishing some species in the wild.",
                paragraph_02:
                  "Historically, cheetahs have not bred well in captivity. In recent years, through cooperative research and management " +
                  "in zoos worldwide, a global master plan is producing successful results. If future extinction were to occur with wild " +
                  "cheetahs, this research and maintained captive populations could allow the re-introduction of cheetahs."
              }
            },
            part_Research: {
              title: "Research",
              content: {
                paragraph_01:
                  "Zoos, universities, private individuals, governmental and NGOs support cooperative efforts worldwide.",
                paragraph_02:
                  "Reproductive, medical and behavioral studies, and laboratory analysis are cooperative conservation efforts between captive and wild cheetah."
              }
            },
            part_Ambassadors: {
              title: "Ambassadors",
              content: {
                paragraph_01:
                  "As representatives of the wild cheetah, zoo animals provide the opportunity for people around the world to see a cheetah close up.",
                paragraph_02:
                  "Through educational programs, zoos spark the interest of individuals who can support or join the projects that will save wild cheetahs.",
                paragraph_03:
                  "Zoos contribute to the conservation of wild cheetahs through education, research and support."
              }
            }
          }
        },
        subsection_SchoolsTeachersLearners: {
          title: "Schools, Teachers, Learners",
          contents: {
            paragraph_01:
              "CCF develops outreach school programs and educational packets are distributed to students and teachers. " +
              "All schools are welcomed to CCF facilities.",
            paragraph_02:
              "Staff members conduct assemblies throughout Namibia. Students are encouraged to become more involved in " +
              "conservation in school and in their homes. Workshops assist teachers in initiating cheetah conservation and " +
              "environmental education programs.",
            paragraph_03:
              "Educational facilities at CCF’s headquarters encourage participation in cheetah conservation. " +
              "The Education Center, the Visitor Center and Predator-Preyground provide information in an interacting setting. " +
              "CCF’s Wildlife Camp allows outdoor education and a wilderness experience. " +
              "A Nature Trail highlights the farmland ecosystem and the valuable role of predators.",
            paragraph_04:
              "CCF takes a cross-curricular approach, integrating conservation issues into subjects that are required as " +
              "part of the school syllabus and donates books to schools and libraries.",
            paragraph_05:
              "CCF works with government agencies to incorporate cheetah conservation in the school curriculum."
          }
        },
        subsection_CommunityEvents: {
          title: "Community Events",
          content: {
            paragraph_01:
              "Participatory community events, such as competitions in art, poetry and writing, increase conservation awareness.",
            paragraph_02:
              "Public educational displays are maintained at strategic locations and community events. " +
              "Displays provide introductory information on the plight of the cheetah and the work conducted by CCF.",
            paragraph_03:
              "CCF staff participate in farmer meetings, local conservancy activities and other related events. " +
              "Workshops and seminars are conducted to exchange information and develop conservation-based farming practices."
          }
        }
      }
    }
  },
};
