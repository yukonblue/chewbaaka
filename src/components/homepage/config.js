/**
 * homepage/config.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 28, 2020
 * Updated  : Aug 17, 2021
 */

export const config = {
  components: {
    landingPageGateway: {
      constants: {
        landingPageGatewayTitle: "Learn about cheetahs",
        landingPageGatewaySubtitle: "Explore the Cheetah Museum at Cheetah Conservation Fund"
      },
    }
  }
}
