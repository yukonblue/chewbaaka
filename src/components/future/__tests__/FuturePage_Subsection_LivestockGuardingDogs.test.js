/**
 * FuturePage_Subsection_LivestockGuardingDogs.test.js
 * Chewbaaka
 *
 * Author   : Tomiko
 * Created  : Jul 22, 2020
 * Updated  : Aug 19, 2021
 */

import React from 'react'

import { render } from '@testing-library/react'

import { config } from '../config'

import renderer from 'react-test-renderer'

import FuturePageSubsectionLivestockGuardingDogs from '../FuturePage_Subsection_LivestockGuardingDogs'

const sectionConfig = config.contentPageSections["section_SustainableDevelopment"];

test('renders FuturePageSubsectionLivestockGuardingDogs component', () => {
  /**
   * Disable test because we currently use require.context
   * and it doesn't work in Jest.
   */
  // render(
  //   <FuturePageSubsectionLivestockGuardingDogs
  //     sectionConfig={sectionConfig}
  //   />
  // );
});

test('FuturePageSubsectionLivestockGuardingDogs component snapshot', () => {
  /**
   * Disable test because we currently use require.context
   * and it doesn't work in Jest.
   */
  // const tree = renderer
  //   .create(
  //     <FuturePageSubsectionLivestockGuardingDogs
  //       sectionConfig={sectionConfig}
  //     />
  //   ).toJSON();
  // expect(tree).toMatchSnapshot();
});
