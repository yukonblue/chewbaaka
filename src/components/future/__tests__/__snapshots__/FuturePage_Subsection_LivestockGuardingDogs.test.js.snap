// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FuturePageSubsectionLivestockGuardingDogs component snapshot 1`] = `
<section
  className="ContentPageSubsectionTemplateOuterContainer CommonRoundedBorderRadiusStyle"
  style={
    Object {
      "backgroundColor": "#F1F0FF",
    }
  }
>
  <h3
    className="ContentPageSubsectionTitle"
  >
    Livestock Guarding Dogs
  </h3>
  <div>
    <div
      className="ContentPageSubsectionPartContainer"
    >
      <h4
        className="ContentPageSubsectionSubtitle"
      >
        CCF’s Livestock Guarding Dog History
      </h4>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        In 1994, CCF created a solution to non-lethal predator control - Livestock Guarding Dogs (LGD). After extensive research, CCF felt that the Anatolian Shepherd and Kangal dogs best met the livestock managers’ needs in Namibia. In cooperation with Dr. Ray Coppinger from the Livestock Guarding Dog Association in Hampshire College (Massachusetts, U.S.), the first 10 dogs were donated to Namibia by Birinci Anatolians.
      </p>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        Since then, hundreds of farmers have received Anatolian Shepherds and Kangal dogs to ward off predators, such as cheetahs, jackals and leopards. These dogs do not herd the goats, but protect the flock by placing themselves between the herd and the predator and barking loudly.
      </p>
      <p
        className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
      >
        CCF’s success is due to the commitment of CCF staff members and the participating farmers. CCF staff regularly visit the farms, educating and advising the farmers and herders for the dogs to be as effective as possible.
      </p>
      <div
        className="VerticalCushionPadding"
      >
        <div
          className="Centered FullWidth"
          style={
            Object {
              "width": 1200,
            }
          }
        >
          <img
            alt="Livestock Guarding Dog"
            className="FullWidth DisplayBlock RetainAspectRatio"
            height={809}
            src="CCF_LGD_L-min.jpg"
            width={1200}
          />
        </div>
      </div>
    </div>
    <div
      className="ContentPageSubsectionPartContainer"
    >
      <h4
        className="ContentPageSubsectionSubtitle"
      >
        Raising LGD’s
      </h4>
      <div
        className="ContentPageSideFloatFluidContainerOuterContainer"
      >
        <div
          className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatRight"
        >
          <div
            className="ImageViewOuterContainer"
            style={
              Object {
                "maxWidth": 720,
              }
            }
          >
            <div
              className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
            >
              <div
                className="ImageViewImgContainer"
              >
                <img
                  alt="Anatolian Shepherd puppies live alongside with the herd."
                  className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                  height={480}
                  src="CCF_anatolian_shepherd_puppies-min.jpg"
                  width={720}
                />
              </div>
              <div
                className="ImageViewFamilyCaptionContainerStyles"
              >
                <div
                  className="CaptionContainer"
                  data-testid="CaptionComponentTestId"
                  style={
                    Object {
                      "maxWidth": 720,
                    }
                  }
                >
                  <div
                    className="IconContainer"
                  >
                    <i
                      aria-hidden="true"
                      className="image icon"
                      onClick={[Function]}
                    />
                  </div>
                  <p
                    className="ContentPageCaptionTextSize CaptionText"
                  >
                    Anatolian Shepherd puppies live alongside with the herd. (Image credit: Cheetah Conservation Fund)
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="ContentPageSideFloatFluidContainerFixedPartContainer"
        >
          <p
            className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
          >
            LGD’s are placed with their herds around nine weeks of age, as social bonding between the stock and the dogs deepens upon imprinting before 16 weeks of age. The dogs live with their livestock family instead of a human family, so the puppies bond with the herd; it becomes the dog’s pack rather than the dog’s potential prey. At CCF puppies are born in a whelping pen attached to the main goat yard.
          </p>
        </div>
      </div>
    </div>
    <div
      className="ContentPageSubsectionPartContainer"
    >
      <div
        className="Centered FullWidth"
        style={
          Object {
            "width": 1000,
          }
        }
      >
        <div
          className="ImageViewOuterContainer"
          style={
            Object {
              "maxWidth": 1000,
            }
          }
        >
          <div
            className="ImageViewInnerContainer ImageViewFamilyContainerBorderStyles ImageViewFamilyContainerShadow ImageViewFamilyContainerBackgroundStyles"
          >
            <div
              className="ImageViewImgContainer"
            >
              <img
                alt="Established in 1994, 2019 marked the 25th anniversary of CCF's Livestock Guarding Dog program."
                className="ImageViewFamilyImgLayoutStyles ImageViewFamilyImgBorderStyles"
                height={299}
                src="CCF_Year_of_the_LGD-min.jpg"
                width={1000}
              />
            </div>
            <div
              className="ImageViewFamilyCaptionContainerStyles"
            >
              <div
                className="CaptionContainer"
                data-testid="CaptionComponentTestId"
                style={
                  Object {
                    "maxWidth": 1000,
                  }
                }
              >
                <div
                  className="IconContainer"
                >
                  <i
                    aria-hidden="true"
                    className="image icon"
                    onClick={[Function]}
                  />
                </div>
                <p
                  className="ContentPageCaptionTextSize CaptionText"
                >
                  Established in 1994, 2019 marked the 25th anniversary of CCF's Livestock Guarding Dog program. (Image credit: Cheetah Conservation Fund)
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ContentPageSubsectionPartContainer"
    >
      <h4
        className="ContentPageSubsectionSubtitle"
      >
        Dogs Save Cheetahs
      </h4>
      <div
        className="ContentPageSideFloatFluidContainerOuterContainer"
      >
        <div
          className="ContentPageSideFloatFluidContainerFloatPartContainer BottomMargin20px FloatRight"
        >
          <img
            alt="Cheetah and LGD"
            className="FullWidth DisplayBlock RetainAspectRatio"
            height={576}
            src="Cheetah_and_LGD-min.jpg"
            width={720}
          />
        </div>
        <div
          className="ContentPageSideFloatFluidContainerFixedPartContainer"
        >
          <p
            className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
          >
            Livestock Guarding Dogs have been used for thousands of years to protect cattle, sheep and goats from predator attacks. There are over 20 breeds of guarding dogs. The Anatolian Shepherd, a Turkish breed, was selected as the best candidate for use with smallstock in Namibia as they are able to work in vast open spaces without direct guidance. These dogs look similar to the flock, with large rounded heads, floppy ears and short fur. They have a good sense of hearing and smelling, a calm temperament and a very loud bark.
          </p>
          <p
            className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
          >
            Anatolians do not herd livestock, they guard them. They are attentive, protective, trustworthy, and aggressive towards predator threats.
          </p>
          <p
            className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
          >
            Puppies are placed with the stock at eight weeks of age to form a strong bond with the herd. It is important that bonding occurs with the herd and not with humans or other dogs.
          </p>
          <p
            className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
          >
            Dog owners take the responsibility for the health care of their dogs. This includes veterinary check-ups, vaccinations, and observations for signs of illness or injuries.
          </p>
          <p
            className="ContentPageRegularTextSize ContentPageRegularTextLineHeight ContentPageContentParagraphText"
          >
            An appropriate diet is necessary for the Anatolian Shepherd. Enough food allows proper growth and a healthy dog. They should never be given raw meat as this could produce a predatory response. They need water during the day when they are out with the herd.
          </p>
        </div>
      </div>
      <div
        className="VerticalCushionPaddingTopLarge"
      >
        <img
          alt="Livestock Guarding Dogs"
          className="FullWidth DisplayBlock RetainAspectRatio Centered"
          height={960}
          src="LGD_960x960_L-min.png"
          width={960}
        />
      </div>
    </div>
  </div>
</section>
`;
